cache/edge_lambdas/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN yarn
 
 RUN zip -r edge_lambda_origin.zip toggler.js origin.js redirector.js redirects.js
 
-CMD ["cp", "/usr/src/app/webapp/edge_lambda_origin.zip", "/dist/.dist/edge_lambda_origin.zip"]
+CMD ["true"]
```

Comment:
What's happening here?

Reply:
This `Dockerfile` for `edge_lambdas` just builds the artefact and hold it in its file system. `docker-compose` then overrides the `CMD` step specified when running tests or deploying.

I don't think it really makes sense for this container to do anything when run without a specific overriding command. 

`true` just makes it run and `exit 0`. This could probably not specify a command at all, but this is consistent with other `Dockerfiles` in this repo that inherit a `CMD` from a parent (and might do something unexpected when run if the command wasn't overridden).

The previous `CMD` was from an experiment with exporting the artefact when the container was run that was discarded, but this wasn't fixed.